packages/core-sdk/lib/Blockchains/OntologyTestnet/Config.ts

```diff
@@ -0,0 +1,36 @@
+import { ONGNativeToken } from "./NativeToken";
+import { Blockchains, EthChainIds, OfflineConnectors } from "../../Types";
+import { MetamaskConnector } from "../../Connectors/Wallets/MetamaskConnector";
+import { blockchainConfigFactory, web3ConnectorFactory } from "../utils";
+import { MetamaskCompatibleConnector } from "../../Connectors/Wallets/MetamaskCompatibleConnector";
+
+export const OntologyTestnetConfig = blockchainConfigFactory(
+  {
+    blockchain: Blockchains.OntologyTestnet,
+    chainId: EthChainIds.OntologyTestnet,
+    publicRpc: "http://polaris1.ont.io",
```

Comment:
This should be `http://polaris1.ont.io:20336`... in fact to avoid mixed content, it should be: `https://polaris1.ont.io:10339`

+ info: https://medium.com/ontologynetwork/everything-you-need-to-know-about-ontology-evm-contract-development-part-1-network-details-2095f6e141e5